projects/graph/src/graph.py

```diff
@@ -74,8 +75,11 @@ def bfs(self, starting_point, value):
             return True
         return False
 
-    def dfs(self):
-        pass
+    def dfs(self, starting_point, value):
+        a = self.dfts(starting_point)
+        if str(value) in a:
+            return True
+        return False
```

Comment:
I'm not sure this is really a depth first search. And I don't think either of these would work very well for an unidrected graph. But this is certainly interesting not sure I've ever seen a solution like this.